### ISSUE TITLE
Support of Jinja, Nunjucks and Swig\Twig

### ISSUE THREAD

**Issue by ArmorDarks** [arendjr/grunt-xgettext#21]
[Jinja](http://jinja.pocoo.org/) and [Nunjucks](https://mozilla.github.io/nunjucks/) are two quite popular templating languages those days. There is also Twig and Swig, which, as far as I know, are pretty close too.

They are almost identical (except Jinja is for Python, while Nunjucks kind of JavaScript fork).

Is there any chances to see support of those templating languages?

Nunjucks (and I assume Jinja too) in fact using function calls, that are exactly JavaScript ones.

So, in templates usually you have something like that:

``` twig
{{ i18n('String to extract', *someOtherParameters*) }}
```

or Jed-like variant, with chaining:

``` twig
{{ i18n('String to extract').ifPluarl('Pluarl string'.fetch(*someOtherParameters*) }}
```

Basically, it means that JavaScript parser could easily traverse through those function. He only needs to be pointed to them, since they located in `{{ }}` or `{% %}` brackets. But, of course, JavaScript parser fails, because it finds some strange expression like `{%` and so on.

The only option for now is to use `html` layout, but it's very limiting, because that works:

``` twig
{{ i18n('String to extract') }}
```

and that doesn't

``` twig
{{ i18n('String to extract', *someOtherParameters*) }}
```

(last variant simply will be omitted)

Thanks in advance!

**Comment by ArmorDarks** [comment]
Is that project still alive?

**Comment by arendjr** [comment]
Sorry no, I don't use this project anymore. I'd be happy to hand it over to someone willing to maintain it.

**Comment by ArmorDarks** [comment]
Sorry to hear that. There are so few tools for extraction of gettext strings for Node.

I'd like to maintain it, but I'm still not good enough with parsers. I'm trying to dig it for quite a long time, but still not there :)

Why not put `[MAINTAINER WANTED]` in description and readme? Hopefully someone will pick it up.

*offtopic:* If it isn't secret, you don't use `gettext` for i18n anymore? For some reason?
